proposals/p0044.md

```diff
@@ -115,9 +115,6 @@ proposals, changes in approval from this doc may affect the evolution process.
 Explicit wording changes aren't outlined here because they are assumed to be
 part of executing this proposal rather than the substance of this proposal.
 
-For reference, a version of this proposal is also available in
```

Comment:
I'd lean towards maintaining the original text, and just commenting out the link. e.g.:

```
For reference, a version of this proposal is also available in
GitHub (broken link: `https://github.com/carbon-language/carbon-proposals/blob/d1946fab824742b4857fa74ffc1b9fe9af37375d/proposals/p0001.md`).
```

Below, for "comment threads" I'd say similar. There's a nuance to the edit where an example of what it's being talked about is being deleted, but I think the example remains helpful in figuring out what it was talking about, even with the broken link.

Reply:
So do we want to use this specific format for all broken links that couldn't be updated?

Reply:
Thanks @jonmeow. I've restored and annotated the broken links that had no replacement.